packages/pwa-kit-dev/src/utils/extensibility-utils.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import path from 'path'
+import fs from 'fs-extra'
+
+const REACT_EXTENSIBILITY_FILE = 'setup-app'
+const SUPPORTED_FILE_TYPES = ['.ts', '.js']
+
+/**
+ * Given a list of extensions, returns an object where the key is the extensions
+ * app entry import string, and the value the path to the source file.
+ *
+ * @param {string[]} extensions - Array of possible file extensions.
+ * @returns {string|null} - The path of the first found file, or null if not found.
+ *
+ * @example
+ * const extensions = ['@salesforce/extension-store-finder', '@salesforce/extension-checkout']
+ *
+ * buildAliases(extensions)
+ * // Output
+ * {
+ *     ['@salesforce/extension-store-finder/setup-app']: '/path/to/setup-app.ts',
+ *     ['@salesforce/extension-checkout/setup-app']: '/path/to/setup-app.ts',
+ * }
+ */
+export const buildAliases = (extensions = []) => {
```

Comment:
super nit, it feels like this utility is building the paths, as oppose to aliases 🤔

Reply:
Can you elaborate on that a little?